src/istgt_lu_disk_vbox.c

```diff
@@ -98,23 +103,25 @@ istgt_lu_disk_open_vbox(ISTGT_LU_DISK *spec, int flags, int mode __attribute__((
 	if (major > 4 || (major == 4 && minor >= 2)) {
 		// VBoxDDU >= 4.2
 		if (ISTGT_VBOXINC_VERSION_MAJOR < 4 ||
-		    (ISTGT_VBOXINC_VERSION_MAJOR == 4 && ISTGT_VBOXINC_VERSION_MINOR < 2)) {
+				(ISTGT_VBOXINC_VERSION_MAJOR == 4 &&
+				ISTGT_VBOXINC_VERSION_MINOR < 2)) {
 			ISTGT_ERRLOG("VBox library is newer than istgt\n");
-			return -1;
+			return (-1);
 		}
 	} else {
 		// VBoxDDU < 4.2
 		if (ISTGT_VBOXINC_VERSION_MAJOR > 4 ||
-		    (ISTGT_VBOXINC_VERSION_MAJOR == 4 && ISTGT_VBOXINC_VERSION_MINOR >= 2)) {
+		    (ISTGT_VBOXINC_VERSION_MAJOR == 4 &&
```

Comment:
as mentioned in previous comment, continuation lines should be given 4 spaces away from the previous line
```
	    (ISTGT_VBOXINC_VERSION_MAJOR == 4 &&
	    ISTGT_VBOXINC_VERSION_MINOR >= 2)) {
```
please check for the same at other code